modules/security-context-constraints-example.adoc

```diff
@@ -18,7 +18,7 @@ allowHostPID: true
 allowHostPorts: true
 allowPrivilegedContainer: true
 allowedCapabilities: <1>
-- '*'
+- KILL
```

Comment:
Because the doc context here is a `privileged` example (it has words `privileged SCC`, `name: privileged` and `description: 'privileged allows`), we'd better let our example YAML indeed consist with `privileged`. Thus, if here it deletes `'*'` and replaces it with `KILL`, the semantics is not about `privileged` any longer.
In a fresh env, `oc get scc privileged -o yaml` shows:
```
...
allowedCapabilities:
- '*'
...
requiredDropCapabilities: null
...
```
Thus I suggest keep `- '*'`, and just remove the `- KILL`, ... and `- SETGID`, and change to `requiredDropCapabilities: null <5>`. After all, the doc's other place has non-null requiredDropCapabilities demo under `For example, to create an SCC that drops the KILL ...`